content/id/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Kustomisasi Service DNS
+content_type: task
+min-kubernetes-server-version: v1.12
+---
+
+<!-- overview -->
+Laman ini menjelaskan cara mengonfigurasi DNS
+{{< glossary_tooltip text="Pod" term_id="pod" >}} kamu dan menyesuaikan
+proses resolusi DNS pada klaster kamu.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Klaster kamu harus menjalankan tambahan (_add-on_) CoreDNS terlebih dahulu.
+[Migrasi ke CoreDNS](/id/docs/tasks/administer-cluster/coredns/#migrasi-ke-coredns)
+menjelaskan tentang bagaimana menggunakan `kubeadm` untuk melakukan migrasi dari `kube-dns`.
+
+{{% version-check %}}
+
+<!-- steps -->
+
+## Pengenalan
+
+DNS adalah Service bawaan dalam Kubernetes yang diluncurkan secara automatis
```

Comment:
https://kbbi.kemdikbud.go.id/entri/otomatis is the formal one, perhaps you got confused by automasi.
```suggestion
DNS adalah Service bawaan dalam Kubernetes yang diluncurkan secara otomatis
```

Reply:
Yes. I do :)